rotkehlchen/tests/unit/test_data_updates.py

```diff
@@ -90,16 +113,39 @@ def test_no_update_performed(data_updater: RotkiDataUpdater) -> None:
     """
     Check that the updates don't execute if we have a higher version locally
     """
-    # set a higer last version applied
+    # set a higher last version applied
     with data_updater.user_db.conn.write_ctx() as write_cursor:
-        write_cursor.execute(
+        write_cursor.executemany(
             'INSERT OR REPLACE INTO settings(name, value) VALUES (?, ?)',
-            (UpdateType.SPAM_ASSETS.serialize(), 999),
+            [
+                (UpdateType.SPAM_ASSETS.serialize(), 999),
+                (UpdateType.RPC_NODES.serialize(), 999),
+            ],
         )
 
     with (
         patch('requests.get', wraps=mock_github_data_response_old_update),
         patch.object(data_updater, 'update_spam_assets') as spam_assets,
+        patch.object(data_updater, 'update_rpc_nodes') as rpc_nodes,
     ):
         data_updater.check_for_updates()
         assert spam_assets.call_count == 0
+        assert rpc_nodes.call_count == 0
+
+
+def test_update_rpc_nodes(data_updater: RotkiDataUpdater) -> None:
+    """Test that rpc nodes are for different blockchain are updated correctly."""
+    data_updater.msg_aggregator.consume_warnings()
```

Comment:
Why consume warnings?

Reply:
oh i copied the template from that of `test_update_spam_assets`

Reply:
so if it's not needed don't do it